examples/llm_inference/android/app/src/main/java/com/example/mediapipe/llminference/ChatUiState.kt

```diff
@@ -0,0 +1,66 @@
+package com.example.mediapipe.llminference
+
+import androidx.compose.runtime.toMutableStateList
+import java.util.UUID
+
+const val START_TURN = "<start_of_turn>"
```

Comment:
Since the IT model requires you to format the prompt in a specific way (e.g. adding user / model prefix), I think it's better to put those logics into a single file rather than multiple files ChatUiState.kt, ChatViewModel.kt and InferenceModel.kt for readability. 

I think having a class like GemmaChat in this notebook would make it easier for our users to follow.
https://colab.sandbox.google.com/drive/1vpsw1KaimcNCJNvQlEqZTqwtDsu3IaZ3?resourcekey=0-8-pLDyn-OWvZMR1kfEIbZQ#scrollTo=O0TkvhOASI9f

Reply:
I moved all the Gemma logic to a `GemmaUiState` class.
Developers using a different model can use `ChatUiState` instead. Both classes implement the same interface to keep them consistent.